### PR TITLE
Removed prefix from animation class name

### PR DESCRIPTION
**Pull request by joaoskr**
With `prefix` var in `animationName` const, this function won't work properly

### CODE REVIEW

**Comment by eltonmesquita** [comment]
Oh, sorry, my mistake. But I'd prefer to fix the other way, on the usage part.

File: docsSource/sections/04-javascript.md [hunk 1]
```diff
@@ -31,7 +31,7 @@ You can also use a simple function to add the animations classes and remove them
 const animateCSS = (element, animation, prefix = 'animate__') =>
   // We create a Promise and return it
   new Promise((resolve, reject) => {
-    const animationName = `${prefix}${animation}`;
```
Comment (eltonmesquita) [inline docsSource/sections/04-javascript.md:34]: Please let this as it is and change the 53 and 57 lines from:
```javascript
animateCSS('.my-element', 'animate__bounce');
```

to:

```javascript
animateCSS('.my-element', 'bounce');
```

**Comment by eltonmesquita** [comment]
Also, please compile the docs running `npm run docs".

**Comment by joaoskr** [comment]
I will open a new PR with this feature then

**Pull request closed without merging**